i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/manage-users/using-management-api.md

```diff
@@ -4,3 +4,27 @@ sidebar_position: 2
 ---
 
 # 使用管理 API 来管理用户
+
+「[管理 API](../../../docs/references/core/#管理-api)」是一个 API 集合，用于管理 Logto 数据，包括用户。
+
+:::info
+调用管理 API 时必须进行 [认证（Authentication）](../../../docs/references/core/#认证authentication)。
+:::
+
+与用户相关的 [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer)
+API 都挂载在 `/api/users`，除了用户活动，即用户日志 `/api/logs?userId=:userId`。
+
+「管理控制台」可以做什么，「管理 API」也可以。
+
+| 管理控制台 - 用户管理                                                                     | 管理 API                                                                                                                                             |
+| :---------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [列出和搜索用户](../../../docs/recipes/manage-users/using-admin-console#列出和搜索用户)   | <a href="/api/#tag/Users/paths/~1api~1users/get" target="_blank"> `GET /api/users`</a>                                                               |
+| [查看用户资料](../../../docs/recipes/manage-users/using-admin-console#查看和更新用户资料) | <a href="/api/#tag/Users/paths/~1api~1users~1:userId/get" target="_blank">`GET /api/users/:userId`</a>                                               |
+| [查看用户活动](../../../docs/recipes/manage-users/using-admin-console#查看用户活动)       | <a href="/api/#tag/Logs/paths/~1api~1logs/get" target="_blank">`GET /api/logs?userId=:userId`</a>                                                    |
```

Comment:
Should we remove this one from this session? It seems it fits in a "logging" session, IMHO.

Reply:
I think the "User activities"/"User logs" is also part of user details.
@logto-io/eng any idea?

Reply:
Keep it per Slack [discussion](https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1656754990926739)